consensus/expected.go

```diff
@@ -84,10 +84,10 @@ const AncestorRoundsNeeded = miner.LargestSectorSizeProvingPeriodBlocks + miner.
 // A Processor processes all the messages in a block or tip set.
 type Processor interface {
 	// ProcessBlock processes all messages in a block.
-	ProcessBlock(ctx context.Context, st state.Tree, vms vm.StorageMap, blk *types.Block, ancestors []types.TipSet) ([]*ApplicationResult, error)
+	ProcessBlock(context.Context, state.Tree, vm.StorageMap, *types.Block, []*types.SignedMessage, []types.TipSet) ([]*ApplicationResult, error)
```

Comment:
We seem to vary as to whether to have param names in an interface declaration.  I waffle on it but generally I think I've decided in the code base I'd prefer it, because so often it clarifies purpose.  Do we have a stated preference?

Reply:
I only think it's better if there are say two big.Ints, or a base type where it's not obvious (e.g. "WaitForSecs(int)" is obvious what the int is for).